src/main/java/modtrekt/logic/commands/AddCommand.java

```diff
@@ -16,41 +16,36 @@ public class AddCommand extends Command {
 
```

Comment:
iirc, our UG specifies that adding module should be done with `add -m <details>` or `add module <details>`, this will conflict with the task portion as it uses `add -t` for now